library/std/src/net/udp.rs

```diff
@@ -159,7 +159,7 @@ impl UdpSocket {
     /// This will return an error when the IP version of the local socket
     /// does not match that returned from [`ToSocketAddrs`].
     ///
-    /// See issue #34202 for more details.
+    /// See [`Issue #34202`] for more details.
```

Comment:
The backticks are only needed for code, to format it monospace font.
```suggestion
    /// See [issue #34202] for more details.
```

Reply:
fixed